tools/development/server.js

```diff
@@ -251,13 +251,13 @@ class HotServers {
         });
         console.log(stats.toString());
         return;
```

Comment:
Doesn't this return statement prevent any of the following lines from executing in the case of an error?